packages/hoprd/hopr-admin/src/utils/command.ts

```diff
@@ -114,12 +167,12 @@ export abstract class Command {
       // validate each parameter
       for (let i = 0; i < params.length; i++) {
         const [paramType] = params[i]
-        const [, validate] = CMD_PARAMS[paramType]
+        const [, , validate] = CMD_PARAMS[paramType]
         const queryParam = queryParams[i]
 
         const [valid, parsedValue] = validate(queryParam, { aliases })
         if (!valid) {
-          result = [`Incorrect parameter "${queryParam}" of type "${paramType}".\n${this.usage()}`, use]
+          result = [this.invalidParameter(queryParam, paramType), use]
           continue
```

Comment:
Isn't this the last statement in the loop? `continue` might not be necessary.

Reply:
fixed in 20f8fc7b72b19ee260d997c6b1453439ed607cc4